actions/subscribers.go

```diff
@@ -414,3 +419,66 @@ func BulkRemoveSubscribers(c *gin.Context) {
 		"message": "We will begin processing the file shortly.",
 	})
 }
+
+func DownloadSubscribersReport(c *gin.Context) {
+	u := middleware.GetUser(c)
+
+	fileName := c.Param("filename")
+
+	report, err := storage.GetReportByFilename(c, fileName, u.ID)
+	if err != nil {
+		c.JSON(http.StatusUnprocessableEntity, gin.H{
+			"message": "Report not found",
+		})
+		return
+	}
+
+	if report.Status == entities.StatusFailed {
+		c.JSON(http.StatusUnprocessableEntity, gin.H{
+			"message": "Failed to generate report, please try again.",
+		})
+		return
+	}
+
+	if report.Status == entities.StatusInProgress {
+		c.JSON(http.StatusUnprocessableEntity, gin.H{
+			"message": "Generating report, please try again later.",
+		})
+		return
+	}
+
+	if report.Status == entities.StatusDone {
+		client, err := awss3.NewS3Client(
+			os.Getenv("AWS_S3_ACCESS_KEY"),
+			os.Getenv("AWS_S3_SECRET_KEY"),
+			os.Getenv("AWS_S3_REGION"),
+		)
+		if err != nil {
+			logger.From(c).WithError(err).Error("Unable to create s3 client.")
+			c.JSON(http.StatusInternalServerError, gin.H{
+				"message": "Unable to sign url.",
+			})
+			return
+		}
+
+		req, _ := client.GetObjectRequest(&s3.GetObjectInput{
+			Bucket: aws.String(os.Getenv("AWS_S3_BUCKET")),
+			Key:    aws.String(fmt.Sprintf("%d/%s,%s/%s", u.ID, "exports", "subscribers", fileName)),
```

Comment:
```suggestion
			Key:    aws.String(fmt.Sprintf("subscribers/export/%d/%s, u.ID, fileName)),
```

We should keep it consistent like in the import functionality: https://github.com/mailbadger/app/blob/c334dfcae71f97c30cc7e4d8f26f63631a719f3a/actions/s3.go#L50